src/pages/beneficiaryDetails/donateForm/DonerInformationForm.tsx

```diff
@@ -45,15 +51,10 @@ export default function DonerInformationForm({
     if (isLoading === "completed") {
       setIsConinue(true);
     }
-  }, [isLoading]);
+  }, [isLoading,setIsConinue]);
 
   return (
-    <div
-      style={{
-        padding: "20px",
-        borderRadius: "20px",
-      }}
-    >
+    <OuterDiv>
       <br />
 
       {formFields.map((val, key) => (
```

Comment:
Please destructure `val` here for use below. And remember in general that array index (referenced as `key` here) should not be used as a React iterator key. Instead I'd recommend you use the formFields `name` as your unique iterator key.